### DISCUSSION TITLE
GitHub Support does not respond after 10 business days

### DISCUSSION THREAD

**Discussion by jiwonz** [community/community#131229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account was flagged and marked as spam, so I posted a ticket to GitHub Support, but it's been 7 business days and I still haven't heard back. Can it really take this long? Or should I wait longer? Or maybe they simply missed my ticket. I'm having trouble working because my account has been flagged, and I can't share the repository, fork it, or authenticate to a third-party app.
![image](https://github.com/community/community/assets/53703001/9d00f53c-552c-43cb-937d-34b102c590ef)

**Comment by Nothing-Works** [comment]
I am having the same issue as you, I've been waiting for 2-3 months. I'm kind of giving up, as no one can help.

**Comment by ghost** [comment]
Was it resolved? How much time did it take overall?
